onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -217,13 +217,22 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
     QuantizeBFP, 1,
     OpSchema()
         .Attr("bfp_type", "The type of BFP - must match with the BFPType enum", AttributeProto::INT)
-        .Attr("block_dims",
-              "Numbers within a bounding box will span across these dimensions."
-              "Any dimension not in this list is the same for all numbers within a bounding box."
-              "As an example, consider a 2D tensor with shape [d0, d1] and block_dims equal to [1]."
-              "Within a bounding box, all elements will be within the same row but will be from different columnns."
+        .Attr("block_dim",
+              "Each bounding box spans this dimension."
+              "Typically, the block dimension corresponds to the reduction dimension of the matrix multipication that "
+              "consumes the output of this operator."
+              "For example, for a 2D matrix multiplication A@W, QuantizeBFP(A) would use block_dim 1 and "
+              "QuantizeBFP(W) would use block_dim 2."
```

Comment:
Do you mean W would use block_dim 0?

Reply:
thanks for pointing this out - will make the change. it should be block_dim 0 and block_dim 1 respectively.